src/features/hotspots/setup/HotspotTxnsSubmitScreen.tsx

```diff
@@ -51,7 +68,20 @@ const HotspotTxnsSubmitScreen = () => {
 
         finalTxn = onboardAssertTxn
       }
-      await submitTxn(finalTxn)
+      const pendingTxn = await submitTxn(finalTxn)
+
+      // Segment track for assert location
+      track(HotspotEvents.ASSERT_LOCATION_SUBMITTED, {
+        pending_transaction: {
```

Comment:
Maker + variant would be interesting

Reply:
I am adding this feature.

Reply:
Maker is added as 'maker_name` property and the variant is added as `hotspot_type` property in the segment track call.